test/repack.bats

```diff
@@ -790,3 +790,24 @@ function teardown() {
 	[ "$numLinesB" -gt "$numLinesA" ]
 	[ "$numLinesC" -gt "$numLinesB" ]
 }
+
+@test "umoci repack (empty diff)" {
+	# Unpack the original image
+	new_bundle_rootfs && BUNDLE_A="$BUNDLE" ROOTFS_A="$ROOTFS"
```

Comment:
If you're not going to use `BUNDLE_A` and `ROOTFS_A` you can drop the `&& ...` section of this line.